docker/test/integration/minifi/core/ImageStore.py

```diff
@@ -84,6 +87,18 @@ def __build_minifi_cpp_image(self):
 
         return self.__build_image(dockerfile)
 
+    def __build_simple_minifi_cpp_image_with_root(self):
```

Comment:
Because of the specific tag, this may need to be renamed to `__build_minifi_cpp_image_for_kind_proxy_with_root` or something similar to contain the intent of use.

Reply:
I have moved the tagging to KindProxy in https://github.com/apache/nifi-minifi-cpp/pull/1270/commits/34bbb4f50d6dda4ced4a0a105611f4d6a99d9089, so now this function does what its name suggests.